Whoaverse/Whoaverse/Scripts/whoaverse.js

```diff
@@ -204,7 +204,119 @@ $(document).ready(function () {
             }
         });
     });
-
+    
+    var key = {};
+
+    // List of the inputs the user use when creating a new discussion
+    var submitInputs = {
+      post: [
+        '#Title',
+        '#Subverse',
+        '#MessageContent'
+      ],
+      reply: [
+        '#CommentContent'
+      ]
+    };
+    
+    document.onkeydown = function(evt) {
```

Comment:
You're right, but there should not be many form fields out at once as each reply form is only added to the DOM after reply is clicked. So that overhead would be minimal as most users would have only two fields open at a time (the top level comment one, which is always shown, and the comment field that they're writing in. I doubt this micro-optimization will make a difference (especially because there are already hundreds of event handlers on the page).

From your other comments, it seems like you are trying to make a whole event managing system for other things, and this might be better for that.